### ISSUE TITLE
[Accessibility]A11y_VSCode_UserSnippets_ScreenReader: NVDA does not read the texts written beside json files.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52040]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated: MAS1.3.1
ScreenReader Version: NVDA2018.2
Mac Version:HighSierra10.13.5 
Jaws Version: 18.0.5038

Repro Steps:
1. Launch VS Code while NVDA is ON.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "User Snippets" from dropdown..

Actual:
NVDA does not read the texts that are written adjacent to json files.Ex- Jade (Pug) , NVDA only reads jade not pug. Issue has also been observed with Jaws and Voiceover.

Expected:
 NVDA does not read the texts written beside json files. Ex-git-commit(Git Commit Message), Jade(Pug). So, suppose Jade is a file, then NVDA does not announce pug or any information about pug.

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
User will not get the full information about snippet files.

MAS Reference:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

Attachment for reference:
[A11y VSCode Problems NVDA Texts adjacent to files.pptx](https://github.com/Microsoft/vscode/files/2106094/A11y.VSCode.Problems.NVDA.Texts.adjacent.to.files.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is a balance between concise information and reading everything.
Feels liek low hanging fruit, assigning to July so we discuss
fyi @bpasero

**Comment by chrmarti** [comment]
@bpasero Should this be done as an opt-in option on the `IconLabel` then? It might make sense for "Open Editors" too.

**Comment by bpasero** [comment]
Yeah and option seems right.
